src/main/java/com/blackducksoftware/integration/fortify/batch/step/BlackDuckFortifyPushThread.java

```diff
@@ -80,8 +78,11 @@
 
     private final static Logger logger = Logger.getLogger(BlackDuckFortifyPushThread.class);
 
+    private SpringConfiguration springConfiguration;
```

Comment:
* Added final for HubServiceFactory and Spring Configuration
* Passed the Spring configuration as a constructor to all required
beans.